changelog/fixed-clear-corrupted-rtt-after-flash

```diff
@@ -0,0 +1 @@
+Fixed corrupted RTT control block clear when download & run.
```

Comment:
Does this actually fix a separate issue to warrant a new changelog? It looks to me that you're just pushing code around for no particular reason.